src/main/java/com/github/wovnio/wovnjava/WovnServletFilter.java

```diff
@@ -93,6 +93,7 @@ private void tryTranslate(Headers headers, RequestOptions requestOptions, HttpSe
         responseHeaders.setApiStatus("Unused");
 
         if (headers.getIsPathInDefaultLanguage()) {
+            WovnLogger.log("Passing to next filter");
```

Comment:
we should keep this log?

Reply:
yes, we otherwise don't have any logging for this branch